test_elasticsearch/test_dsl/conftest.py

```diff
@@ -49,6 +52,54 @@
 )
 
 
+def pytest_configure(config: "pytest.Config"):
```

Comment:
Did you consider using two fixtures instead? This would be more explicit.

Reply:
Okay, I moved these mocks inside the actual test function, so they are now local to the one test that needs them. I've also switched to `mock.patch.dict()` to mock `sys['modules']`, which results in much cleaner code. WDYT?